gum/backend-linux/gumandroid.c

```diff
@@ -598,13 +600,16 @@ gum_try_init_linker_details (void)
       break;
     }
   }
+
```

Comment:
```suggestion
```

(To keep it part of the same "operation".)

Reply:
Inadvertent when I was moving it around, good catch.